spec/views/login_spec.rb

```diff
@@ -0,0 +1,54 @@
+# rubocop:disable Metrics/BlockLength
```

Comment:
Please don't disable linter rules as this makes code clean.

Reply:
@arslanbisharat thank you for your review 🙏. Fixed